### PR TITLE
allow websocket proxying

### PR DESCRIPTION
**Pull request by pklingem**
/cc @tecnobrat

### CODE REVIEW

File: nginx/templates/containers.conf [hunk 1]
```diff
@@ -15,6 +15,11 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header Host $host;
     proxy_set_header X-Real-IP $remote_addr;
+
+    # Websocket
+    proxy_http_version 1.1;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection $connection_upgrade;
```
Comment (tecnobrat) [inline nginx/templates/containers.conf:22]: Need this in the ssl version too

**Comment by pklingem** [comment]
@tecnobrat added for SSL as well.

**Comment by tecnobrat** [comment]
Looks good. :shipit:

Can you make a new release and bump tugboat-bootstrapper too?

**Comment by pklingem** [comment]
glad ta

**Pull request merged**